main.go

```diff
@@ -46,32 +52,32 @@ func main() {
 
 	istioClient, err := crd.NewClient(*kubeconfig, "", configDescriptor, *dnsSuffix)
 	if err != nil {
-		log.Panicln("Error creating istio crd client:", err.Error())
+		log.Panicf("%s Error creating istio crd client: %s", logPrefix, err.Error())
```

Comment:
it would be nice to have a `log.GetPrefixedLogger(prefix string)` that would be a wrapped logger with the prefix, but not sure if possible with our log package implementation